kotsadm/web/src/components/gitops/GitOpsDeploymentManager.jsx

```diff
@@ -113,21 +101,57 @@ class GitOpsDeploymentManager extends React.Component {
     }
   }
 
+  getGitopsState = async () => {
+    const freshGitops = await this.getGitops()
+
+    if (freshGitops.enabled) {
+      const selectedService = find(SERVICES, service => service.value === freshGitops.provider);
+      this.setState({
+        selectedService: selectedService ? selectedService : this.state.selectedService,
+        hostname: freshGitops.hostname || "",
+        gitops: freshGitops
+      })
+    } else {
+      this.setState({
+        gitops: freshGitops
+      })
+    }
+  }
+
+  getGitops = async () => {
+    try {
+      const res = await fetch(`${window.env.API_ENDPOINT}/gitops/get`, {
+        headers: {
+          "Authorization": Utilities.getToken(),
+          "Content-Type": "application/json",
+        },
+        method: "GET",
+      });
+      if (!res.ok) {
```

Comment:
should log out on `401` status, example: https://github.com/replicatedhq/kots/blob/5a6baa27c54e10529b88f73c860f127f64ff8ee8/kotsadm/web/src/Root.jsx#L151